lnd.go

```diff
@@ -119,13 +119,32 @@ func AdminAuthOptions() ([]grpc.DialOption, error) {
 	return opts, nil
 }
 
+type GrpcRegisterer interface {
+	RegisterGrpcSubserver(*grpc.Server) error
+}
+
+type RestRegisterer interface {
+	RegisterRestSubserver(context.Context, *proxy.ServeMux, string,
+		[]grpc.DialOption) error
+}
+
+type SubserverConfig struct {
+	Permissions map[string][]bakery.Op
+	
+	GrpcRegisterer GrpcRegisterer
```

Comment:
Does this need to be a slice? To register multiple sub-servers? Or is the idea that this method (the concrete implementation), would register them all at once?

Reply:
Yes. This is the same definition as we use for all permissions in `rpcserver.go`. For example if you want to require `onchain` and `offchain` permissions, you pass in two tuples per URI.